experimental/handoff-library/csharp_dotnetcore/Microsoft.Bot.Builder.Handoff.Test/HandoffTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Handoff;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Schema;
+using Microsoft.Rest;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Bot.Builder.Handoff.UnitTests
+{
+    [TestClass]
```

Comment:
We are moving to xunit. Though we still have lots of test in ms test.

Reply:
not changed